### PR TITLE
update: module/base and remove unused  dependencies

### PR DESCRIPTION
**Pull request by alex-Arc**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,9 +23,8 @@
 		"node": ">=18.12.0"
 	},
 	"dependencies": {
-		"@companion-module/base": "~1.2.1",
+		"@companion-module/base": "^1.7.0",
```
Comment (kellhogs) [inline package.json:26]: Is this the latest version of the module base?
If so, did you check that the current stable branch of companion supports this module base version?

I had this issue in the past that the latest module base isn't supported by the current stable branch of companion.

**Review by kellhogs — COMMENTED** [review]
Just one little question.

File: package.json (hunk 1 above)
Comment (alex-Arc) [reply package.json:26]: if I understand correctly our next release will be part of v3.2 ? 
image

Comment (kellhogs) [reply package.json:26]: Ah yeah, I haven't seen this slack message yet. I just wanted to make sure you checked that this module base version is compatible with the release.

**Pull request merged**